Library/Homebrew/dev-cmd/bump-cask-pr.rb

```diff
@@ -89,6 +89,20 @@ def bump_cask_pr
     odie "This cask is not in a tap!" if cask.tap.blank?
     odie "This cask's tap is not a Git repository!" unless cask.tap.git?
 
+    autobump_file_path = cask.tap.path/Tap::HOMEBREW_TAP_AUTOBUMP_FILE
+    if cask.tap.core_cask_tap? &&
+       autobump_file_path.exist? &&
+       autobump_file_path.readlines(chomp: true).include?(cask.token) &&
+       ENV.fetch("HOMEBREW_TEST_BOT_AUTOBUMP", nil).blank?
+      odie <<~EOS
+        Whoops, the #{cask.token} cask has its version update
+        pull requests automatically opened by BrewTestBot!
+        We'd still love your contributions, though, so try another one
+        that's not in the autobump list:
+          #{Formatter.url("#{cask.tap.remote}/blob/master/.github/autobump.txt")}
+      EOS
+    end
+
```

Comment:
```suggestion
    if ENV.fetch("HOMEBREW_TEST_BOT_AUTOBUMP", false).blank?
      if cask.tap.core_cask_tap? &&
         (autobump_file_path = cask.tap.path/Tap::HOMEBREW_TAP_AUTOBUMP_FILE) &&
         autobump_file_path.exist? &&
         autobump_file_path.readlines(chomp: true).include?(cask.token)
        odie <<~EOS
          Whoops, the #{cask.token} cask has its version update
          pull requests automatically opened by BrewTestBot!
          We'd still love your contributions, though, so try another one
          that's not in the autobump list:
            #{Formatter.url("#{cask.tap.remote}/blob/master/.github/autobump.txt")}
        EOS
      end
    end
```

let's avoid reading from the file etc. when `HOMEBREW_TEST_BOT_AUTOBUMP` is set